approved/v93k/custom_tests.tf

```diff
@@ -1,81 +1,91 @@
 hp93000,testflow,0.1
 language_revision = 1;
- 
-information
- 
- 
-end
---------------------------------------------------
-implicit_declarations
 
-end
------------------------------------------------------------------
 testmethodparameters
+
 tm_1:
-  "testerState" = "CONNECTED";
-  "testName" = "Functional";
+  "BadPractice" = "NO";
```

Comment:
What is this here?

Reply:
I think it's correct.  It's just re-ordering the parameters alphabetically, but the diff makes it look weird.